[2024-11-22T09:02:50.330+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-22T09:02:50.337+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_stock_prediction.load_data_postgres manual__2024-11-22T09:02:37.743470+00:00 [queued]>
[2024-11-22T09:02:50.341+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_stock_prediction.load_data_postgres manual__2024-11-22T09:02:37.743470+00:00 [queued]>
[2024-11-22T09:02:50.341+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-22T09:02:50.349+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_data_postgres> on 2024-11-22 09:02:37.743470+00:00
[2024-11-22T09:02:50.358+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=20161) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-22T09:02:50.360+0000] {standard_task_runner.py:72} INFO - Started process 20184 to run task
[2024-11-22T09:02:50.360+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_stock_prediction', 'load_data_postgres', 'manual__2024-11-22T09:02:37.743470+00:00', '--job-id', '1517', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpnlojg9q3']
[2024-11-22T09:02:50.361+0000] {standard_task_runner.py:105} INFO - Job 1517: Subtask load_data_postgres
[2024-11-22T09:02:50.400+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_stock_prediction.load_data_postgres manual__2024-11-22T09:02:37.743470+00:00 [running]> on host cba1e82f0ade
[2024-11-22T09:02:50.446+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_stock_prediction' AIRFLOW_CTX_TASK_ID='load_data_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-11-22T09:02:37.743470+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-22T09:02:37.743470+00:00'
[2024-11-22T09:02:50.447+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-22T09:02:50.448+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-22T09:02:50.450+0000] {logging_mixin.py:190} INFO - Current task name:load_data_postgres state:running start_date:2024-11-22 09:02:50.338216+00:00
[2024-11-22T09:02:50.450+0000] {logging_mixin.py:190} INFO - Dag name:weather_stock_prediction and current dag run status:running
[2024-11-22T09:02:50.450+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-22T09:02:50.482+0000] {base.py:84} INFO - Retrieving connection 'asdf'
[2024-11-22T09:02:50.483+0000] {sql.py:544} INFO - Running statement: 
    INSERT INTO weatherstocks (date, temperature, min_temperature, max_temperature, humidity, wind_speed, price)
    VALUES (%s, %s, %s, %s, %s, %s, %s)
    , parameters: (Timestamp('2024-11-18 00:00:00'), 15.7, 14.58, 17, 83, 4.92, 228.52)
[2024-11-22T09:02:50.485+0000] {sql.py:553} INFO - Rows affected: 1
[2024-11-22T09:02:50.485+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-22T09:02:50.488+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-22T09:02:50.489+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_stock_prediction, task_id=load_data_postgres, run_id=manual__2024-11-22T09:02:37.743470+00:00, execution_date=20241122T090237, start_date=20241122T090250, end_date=20241122T090250
[2024-11-22T09:02:50.491+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-22T09:02:50.492+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-22T09:02:50.492+0000] {logging_mixin.py:190} INFO - Dag name:weather_stock_prediction queued_at:2024-11-22 09:02:37.764521+00:00
[2024-11-22T09:02:50.492+0000] {logging_mixin.py:190} INFO - Task hostname:cba1e82f0ade operator:PythonOperator
[2024-11-22T09:02:50.503+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-22T09:02:50.518+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-22T09:02:50.519+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
